### ISSUE TITLE
Radio groups show multiple selections when use_name_attributes set to false.

### ISSUE THREAD

**Issue by mark-lancaster** [json-editor/json-editor#1171]
### General information

Radio groups show multiple selections when **use_name_attributes** set to **false**.

- json-editor version: 2.8

### Expected behavior

As per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio :

> A radio group is defined by giving each of radio buttons in the group the same [name](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-name). Once a radio group is established, selecting any radio button in that group automatically deselects any currently-selected radio button in the same group.

Radio-groups must always provide a name value, ignoring the use_name_attributes setting.

If the desired behavior is to prevent editor inputs being submitted, then it would be better to disable the inputs within the editors' div container in an on-submit event.

As per https://www.w3.org/TR/html401/interact/forms.html#h-17.13.2 :

> A successful control is "valid" for submission. Every successful control has its [control name](https://www.w3.org/TR/html401/interact/forms.html#control-name) paired with its [current value](https://www.w3.org/TR/html401/interact/forms.html#current-value) as part of the submitted [form data set](https://www.w3.org/TR/html401/interact/forms.html#form-data-set). A successful control must be defined within a [FORM](https://www.w3.org/TR/html401/interact/forms.html#edef-FORM) element and must have a [control name.](https://www.w3.org/TR/html401/interact/forms.html#control-name)
> 
> However:
> 
> Controls that are [disabled](https://www.w3.org/TR/html401/interact/forms.html#disabled) cannot be successful.
> If a form contains more than one [submit button](https://www.w3.org/TR/html401/interact/forms.html#buttons), only the activated submit button is successful.
> All "on" [checkboxes](https://www.w3.org/TR/html401/interact/forms.html#checkbox) may be successful.
> For [radio buttons](https://www.w3.org/TR/html401/interact/forms.html#radio) that share the same value of the [name](https://www.w3.org/TR/html401/interact/forms.html#adef-name-INPUT) attribute, only the "on" radio button may be successful.
> ...

### Actual behavior

Radio group shows multiple selections, but returns single value.

### Steps to reproduce the behavior

Either view the ../tests/pages/use-name-attributes.html or construct a page with the following config and adjust the script path:

```html
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8"/>
    <title>Use name attributes</title>
  <script src="../../dist/jsoneditor.js"></script>
</head>
<body>
  <div class='container'></div>
  <script>
    var schema = {
        "type": "object",
        "title": "root",
        "properties": {
            "gender": {
                "type": "string",
                "format": "radio",
                "title": "Gender",
                "enum": ["male", "female", "unknown"]
            }
        }
    }

    var editor = new JSONEditor(document.querySelector('.container'), {
      use_name_attributes: false,
      schema: schema
    })
  </script>
</body>
</html>

```

**Comment by schmunk42** [comment]
@germanbisurgi Example: https://is.gd/54B3tM

**Comment by germanbisurgi** [comment]
Fixed in #1490
